emanifest-py/src/emanifest/client.py

```diff
@@ -906,3 +832,32 @@ def __RCRAGet(self, endpoint):
             return resp.json()
         else:
             print('Error: ' + str(resp.json()['message']))
+            
+    def __RCRAPost(self, endpoint, **kwargs):
+        resp = requests.post(endpoint,
+                            headers={'Content-Type': 'text/plain', 'Accept': 'application/json',
+                                     'Authorization': 'Bearer ' + self.token},
+                            data=json.dumps(dict(**kwargs)))
+        if resp.ok:
+            return resp.json()
+        else:
+            print('Error: ' + str(resp.json()['message']))
+                   
+    def __RCRADelete(self, endpoint):
+        resp = requests.delete(endpoint,
+                            headers={'Accept': 'application/json','Authorization': 'Bearer ' + self.token})
+        if resp.ok:
+            return resp.json()
+        else:
+            print('Error: ' + str(resp.json()['message']))
+            
+            
+    def __RCRAPut(self, endpoint):
+        resp = requests.put(endpoint,
+                            headers={'Content-Type': m.content_type, 'Accept': 'application/json',
```

Comment:
I think the object "m" here is an undefined reference

Reply:
Good catch - forgot to add that to the parameters. I've updated the function to require it.